CesiumGltf/include/CesiumGltf/PropertyTablePropertyView.h

```diff
@@ -304,30 +304,27 @@ class PropertyTablePropertyView<ElementType, false>
    * @return The value of the element, or std::nullopt if it matches the "no
    * data" value
    */
-  std::optional<ElementType> get(int64_t index) const noexcept {
+  std::optional<PropertyValueViewToCopy<ElementType>>
```

Comment:
Do you need to change the `getRaw` functions to return `PropertyValueViewToCopy`, as well?

Reply:
Hm, maybe not. I got thrown off when I saw `PropertyValueViewToCopy<ElementType>` in `PropertyTexturePropertyView`, but we're actually constructing the values there. Here we don't have to do that.

Reply:
Yeah the value returned by `PropertyTablePropertyView::getRaw` is never transformed, so we can always provide it as a view. Where `PropertyTexturePropertyView` transforms even the raw values is returns.